client/views/rename/rename.js

```diff
@@ -23,9 +23,9 @@ Template.rename.events({
     let newDesc = document.getElementById('descriptionbox').value;
     newDesc = newDesc.charAt(0).toUpperCase() + newDesc.slice(1);
     newDesc = UniHTML.purify(newDesc, { withoutTags: ['a', 'img', 'ol', 'ul', 'span', 'br', 'table', 'caption', 'col', 'colgroup', 'tbody', 'td', 'tfoot', 'th', 'thread', 'tr', 'li'] });
-    if (newName === table.tablename && newDesc === table.description) {
-      return false;
-    }
+    // if (newName === table.tablename && newDesc === table.description) {
```

Comment:
Why is this code commented out instead of removed?

Reply:
Because I wasn't sure about removing that code without getting it reviewed. 
Done in my latest commit.